packages/cxl-ui/src/components/partials/cxl-single-multiversion-card-display.js

```diff
@@ -0,0 +1,106 @@
+import {html} from 'lit-html';
+import {unsafeHTML} from 'lit-html/directives/unsafe-html';
+
+const CXLSingleMultiversionCardDisplay = (el, userId, selected) => {
+  const postType = el.cxl_hybrid_attr_post['@attributes'].class.includes(
+    'category-minidegree-programs'
+  )
+    ? 'minidegree'
+    : 'playbook';
+
+  return html`
+    <cxl-accordion-multiversion-card
+      id="${el.cxl_hybrid_attr_post['@attributes'].id}"
+      class="${el.cxl_hybrid_attr_post['@attributes'].class}"
+      theme="${postType === 'minidegree' ? 'dark' : ''}"
+    >
+      <header class="entry-header" slot="summary">
+        <label title="${postType === 'minidegree' ? 'Course' : 'Playbook'}"
+               class="entry-type">${postType === 'minidegree' ? 'COURSE' : 'PLAYBOOK'}</label>
+
+        <cxl-save-favorite
+          postType="${postType === 'minidegree' ? 'minidegree' : 'playbook'}"
+          postId="${el.cxl_hybrid_attr_post['@attributes'].id}"
+          userId="${userId}"
+          ?selected=${selected}
+          isCardVersion
+        ></cxl-save-favorite>
+
+        <div class="entry-summary" itemprop="description">${el.breadcrumbs.join('/')}</div>
+
+        <h2 class="entry-title no-anchor" itemprop="headline">
+          <a href="${el.conversionxl_certificate_sales_page}" rel="bookmark" itemprop="url"
+          >${el.title.raw}</a
+          >
+        </h2>
+
+        <div class="entry-byline">
+          ${el.versions.length === 1
+            ? html`
+              <div>${el.versions[0].upvotes} votes</div>
+              <div>Author: ${el.versions[0].author}</div>`
+            : html``}
+          ${el.versions.length > 1
+            ? html`
+              <div>
+                Versions: ${html`<span class="author">${el.versions.length}</span>`}
+              </div>`
+            : html``}
```

Comment:
Why is it named `versions` and later it displays `upvotes` and `author`? It doesn't make sense